module/notifications/fcm/helper.js

```diff
@@ -0,0 +1,285 @@
+/**
+ * name : module/notifications/fcm/helper.js
+ * author : Aman Jung Karki
+ * Date : 15-Nov-2019
+ * Description : FCM helper.
+ */
+
+// dependencies
+const NOTIFICATION_COLOR = 
+gen.utils.checkIfEnvDataExistsOrNot("SAMIKSHA_THEME_COLOR");
+const userExtensionsHelper = require(MODULES_BASE_PATH + "/user-extension/helper");
+const firebaseHelper = require(ROOT_PATH + "/generics/helpers/fcm");
+
+/**
+    * FcmHelper
+    * @class
+*/
+
+module.exports = class FcmHelper {
```

Comment:
@priyanka-TL - Why is this new file created? One such file was already there no?

Cc - @aman-tunerlabs

Reply:
@aks30  im deleted that file already dont know who added i will delete it again